data/json/bionics.json

```diff
@@ -523,6 +845,10 @@
         "type": "bionic",
         "id": "bio_power_armor_interface_mkII",
         "name": "Power Armor Interface Mk. II",
+        "occupied_bodyparts": [
+            [ "TORSO", 3 ],
+            [ "HEAD", 1 ]
```

Comment:
It's not less, it's 2+2 vs. 3+1 (just for variety, in some cases you will not migrate from one to another due to lack of one of bodyparts). BTW, it's probably better to switch the numbers and require 2+2 slots for MkII version: head slots "worth" more than "torso" one.